Templates/Integrations/AnalyticsAndSIEM/AnalyticsAndSIEM_test.py

```diff
@@ -0,0 +1,116 @@
+""" HELPER FUNCTIONS """
+from pytest import raises
+from AnalyticsAndSIEM import Client
+
+CONTEXT_PREFIX = 'AnalyticsAndSIEM.Event(val.ID && val.ID === obj.ID)'
+
+
+def event_dict_input(*args, **kwargs):
+    return {'event': {'eventId': 'ab123',
+                      'description': 'Phishing email',
+                      'createdAt': '2010-01-01T00:00:00Z',
+                      'isActive': True,
+                      'assignee': [{'name': 'DBot Demisto', 'id': '11'},
+                                   {'name': 'Demisto DBot', 'id': '12'}]}}
+
+
+def event_dict_output(*args, **kwargs):
+    return {'Event': {'Assignee': [{'ID': '11', 'Name': 'DBot Demisto'},
+                                   {'ID': '12', 'Name': 'Demisto DBot'}],
+                      'Created': '2010-01-01T00:00:00Z',
+                      'Description': 'Phishing email',
+                      'ID': 'ab123',
+                      'IsActive': True}}
+
+
+def event_list_input(*args, **kwargs):
+    return {'event': [{'eventId': 'ab123',
+                       'description': 'Phishing email',
+                       'createdAt': '2010-01-01T00:00:00Z',
+                       'isActive': True,
+                       'assignee': [{'name': 'DBot Demisto', 'id': '11'},
+                                    {'name': 'Demisto DBot', 'id': '12'}]},
+                      {'eventId': 'ab123',
+                       'description': 'Phishing email',
+                       'createdAt': '2010-01-01T00:00:00Z',
+                       'isActive': True,
+                       'assignee': [{'name': 'DBot Demisto', 'id': '11'},
+                                    {'name': 'Demisto DBot', 'id': '12'}]}]}
+
+
+def event_list_output(*args, **kwargs):
+    return {'Event': [{'Assignee': [{'ID': '11', 'Name': 'DBot Demisto'},
+                                    {'ID': '12', 'Name': 'Demisto DBot'}],
+                       'Created': '2010-01-01T00:00:00Z',
+                       'Description': 'Phishing email',
+                       'ID': 'ab123',
+                       'IsActive': True},
+                      {'Assignee': [{'ID': '11', 'Name': 'DBot Demisto'},
+                                    {'ID': '12', 'Name': 'Demisto DBot'}],
+                       'Created': '2010-01-01T00:00:00Z',
+                       'Description': 'Phishing email',
+                       'ID': 'ab123',
+                       'IsActive': True}]}
+
+
+def mock_client() -> Client:
+    return Client(
+        'http://api.service.com',
```

Comment:
the original leads to this:
![image](https://user-images.githubusercontent.com/30797606/64449192-0ff1b680-d0e8-11e9-999a-6a0b5191a5c2.png)

```suggestion
        'http://api.example.com',
```